templates/cluster-template.yaml

```diff
@@ -16,6 +234,35 @@ stringData:
     \       \"username\": \"${NUTANIX_USER}\", \n        \"password\": \"${NUTANIX_PASSWORD}\"\n
     \     }\n    }\n  }\n]\n"
 ---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: nutanix-ccm-secret
+stringData:
+  nutanix-ccm-secret.yaml: "apiVersion: v1\nkind: Secret\nmetadata:\n  name: nutanix-creds\n
+    \ namespace: kube-system\nstringData:\n  credentials: |\n    [\n      {\n        \"type\":
+    \"basic_auth\", \n        \"data\": { \n          \"prismCentral\":{\n            \"username\":
+    \"${NUTANIX_USER}\",\n            \"password\": \"${NUTANIX_PASSWORD}\"\n          },\n
+    \         \"prismElements\": null\n        }\n      }\n    ]\n"
+type: addons.cluster.x-k8s.io/resource-set
+---
+apiVersion: addons.cluster.x-k8s.io/v1beta1
+kind: ClusterResourceSet
```

Comment:
Since this is the default now lets add this to `clusterctl.yaml`:
```
EXP_CLUSTER_RESOURCE_SET: "true"
```

Reply:
It's already there for E2E tests.

Reply:
Thanks yes I see the e2e changes, but we should still change

Reply:
Thanks yes I see the e2e changes, but we should still change

Reply:
@dkoshkin Can we change it in separate PR as soon as it is developer tool, not for end users? Looks like there are bunch of dev tools improvement changes.

Reply:
👍that works for me